qbft/instance.go

```diff
@@ -56,16 +57,26 @@ func (i *Instance) ForceStop() {
 }
 
 // Start is an interface implementation
-func (i *Instance) Start(value []byte, height Height) {
+func (i *Instance) Start(cdFetcher *types.DataFetcher, height Height, valueCheckF ProposedValueCheckF) {
```

Comment:
Can't we use the `i.GetConfig().GetValueCheck()` instead of using an argument? It seems a duplication to me

Reply:
I agree, if value check only used here then maybe we don't need it as part of `Config`.